nifi-nar-bundles/nifi-avro-bundle/nifi-avro-processors/src/test/java/org/apache/nifi/processors/avro/TestConvertAvroToJSON.java

```diff
@@ -239,6 +239,37 @@ public void testSingleSchemalessAvroMessage_wrapSingleMessage_noContainer() thro
     }
 
     @Test
+    public void testSingleSchemalessAvroMessage_wrapSingleMessage_noContainer_StandardJson() throws IOException {
```

Comment:
The test method name says StandardJson but the USE_AVRO_JSON property is set to true in the test